### PR TITLE
TELCODOCS-268: Release note

### PR DESCRIPTION
**Pull request by amolnar-gh**
For 4.9: [Jira issue](https://issues.redhat.com/browse/TELCODOCS-268)

[Release note](https://deploy-preview-36368--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-networking-CNI-VRF-plug-in)

[Technology Preview table](https://deploy-preview-36368--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-technology-preview)

### CODE REVIEW

**Comment by amolnar-rh** [comment]
@fedepaol Could you please review the release note? (The main content is not merged yet)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -116,6 +116,12 @@ For this release, Red Hat only supports using MetalLB in layer 2 mode.
 
 For more information, see xref:../networking/metallb/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator].
 
+[id="ocp-4-9-networking-CNI-VRF-plug-in"]
+==== CNI VRF plug-in
```
Comment (ogradyp) [inline release_notes/ocp-4-9-release-notes.adoc:120]: I referenced previous versions, 4.7 & 4.8, of the release notes, and when a feature is generally available it is included in the title. So I would rename this to 'CNI VRF plug-in is generally available'

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -116,6 +116,12 @@ For this release, Red Hat only supports using MetalLB in layer 2 mode.
...
+The CNI VRF plug-in was previously introduced as a Technology Preview feature in {product-title} 4.8, and is now generally available in {product-title} {product-version}.
```
Comment (ogradyp) [inline release_notes/ocp-4-9-release-notes.adoc:121]: Just one thing I noticed regarding the version number. In this sentence is says the Technology Preview feature was introduced in 4.8, but in Table 2. Technology Preview tracker TP is also included in the OCP 4.7 column. So either this sentence needs to be changed to 4.7, or in Table 2. Technology Preview tracker TP needs to be removed from the OCP 4.7 column.

**Review by ogradyp — COMMENTED** [review]
Peer-review feedback from Padraig.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (cgoncalves) [reply release_notes/ocp-4-9-release-notes.adoc]: +1. The VRF code support was added in OCP 4.7.
https://github.com/openshift/containernetworking-plugins/tree/release-4.7/plugins/meta/vrf
https://issues.redhat.com/browse/CNF-846

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -135,6 +135,13 @@ For this release, Red Hat only supports using MetalLB in layer 2 mode.
 
 For more information, see xref:../networking/metallb/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator].
 
+[id="ocp-4-9-networking-CNI-VRF-plug-in"]
+==== CNI VRF plug-in is generally available
+
+The CNI VRF plug-in was previously introduced as a Technology Preview feature in {product-title} 4.7, and is now generally available in {product-title} {product-version}.
```
Comment (EricPonvelle) [inline release_notes/ocp-4-9-release-notes.adoc:141]: Remove the comma after "{product-title} 4.7"

**Review by EricPonvelle — APPROVED** [review]
/lgtm

One minor correction with a comma.

**Pull request merged**